sdk/attestation/attestation/package.json

```diff
@@ -0,0 +1,100 @@
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/attestation.d.ts",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "devDependencies": {
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/test-utils-recorder": "^1.0.0",
+    "@microsoft/api-extractor": "7.7.11",
+    "@opentelemetry/api": "^0.10.2",
+    "@types/chai": "^4.1.6",
+    "@types/chai-as-promised": "^7.1.0",
+    "@types/jsrsasign": "^8.0.9",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^8.0.0",
+    "buffer": "^5.2.1",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "dotenv": "^8.2.0",
+    "jsrsasign": "^10.1.4",
+    "karma": "^5.1.0",
+    "karma-chrome-launcher": "^3.0.0",
+    "karma-coverage": "^2.0.0",
+    "karma-edge-launcher": "^0.4.2",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-json-preprocessor": "^0.3.3",
+    "karma-json-to-file-reporter": "^1.0.1",
+    "karma-junit-reporter": "^2.0.1",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-remap-istanbul": "^0.6.0",
+    "karma-source-map-support": "~1.4.0",
+    "mocha": "^7.1.1",
+    "prettier": "^1.16.4",
+    "rimraf": "^3.0.0",
+    "rollup": "^1.16.3",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "safe-buffer": "^5.2.1",
+    "ts-node": "^8.3.0",
+    "typescript": "4.1.2"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Azure/azure-sdk-for-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/attestation.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "scripts": {
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
+    "build:samples": "echo skipped",
+    "build:test": "tsc -p . && rollup -c 2>&1",
+    "clean": "rimraf dist dist-esm dist-test temp types *.tgz *.log",
+    "extract-api": "api-extractor run --local",
+    "lint": "echo skipped",
+    "prepack": "npm install && npm run build",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/{,!(browser)/**/}*.spec.ts\"",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "sideEffects": false,
+  "autoPublish": true,
+  "browser": {
+    "./dist-esm/test/utils/base64.js": "./dist-esm/test/utils/base64.browser.js",
+    "./dist-esm/test/utils/base64url.js": "./dist-esm/test/utils/base64url.browser.js",
+    "./dist-esm/test/utils/Buffer.js": "./dist-esm/test/utils/Buffer.browser.js",
+    "./dist-esm/test/utils/env.js": "./dist-esm/test/utils/env.browser.js"
```

Comment:
Can this be reverted now that you get that from the recorder?

Reply:
I am thinking of leaving it there for future-proofing. There might be test suites that we do not want to record but we still want to read env vars for. There is such case in text analytics.

However `base64` one could be replaced by the more general buffer one.